src/lib/loaders/loaders_without_authentication/convection.ts

```diff
@@ -0,0 +1,35 @@
+import config from "config"
+import fetch from "node-fetch"
+import urljoin from "url-join"
+
+const { GRAVITY_XAPP_TOKEN, CONVECTION_API_BASE } = config
+
+export const convectionLoaders = (_opts) => {
+  const createConsignmentInquiryLoader = async <T = unknown>(
+    variables
+  ): Promise<Record<string, T>> => {
+    const body = JSON.stringify(variables)
+
+    const response = await fetch(
+      urljoin(CONVECTION_API_BASE, "consignment_inquiries"),
+      {
+        method: "POST",
+        body,
+        headers: {
+          Authorization: `Bearer ${GRAVITY_XAPP_TOKEN}`,
```

Comment:
I'm a bit confused about this one so wanted to double-check. `GRAVITY_XAPP_TOKEN` is an app token for the Metaphysics client app, signed with the Metaphysics secret. Passing it to Impulse at first glance seems odd, since Impulse wouldn't be able to _verify_ this token, as it doesn't know the Metaphysics secret.

Does that imply that there isn't any authorization on this endpoint in Convection at all (since this token being passed won't be able to be verified)? Would love to sync to connect the dots.

Reply:
The endpoint has authorization and the requirement is a trusted app or user. I thought `GRAVITY_XAPP_TOKEN` is the same across services and I have been testing with the Gravity xapp token as set on convection. This comment made me realise that the token I have been using may be different from this `GRAVITY_XAPP_TOKEN`.
What I need to do is have a `CONVECTION_TOKEN` set in the config and use that instead.